### PR TITLE
HTML-262 Fixing the disallow duplicate encounter submission tag Javascript tag

### PR DESCRIPTION
**Pull request by ssmusoke**
@dkayiwa I am running into trouble with the unit tests do advise

### CODE REVIEW

**Comment by dkayiwa** [comment]
change from legacyui-omod dependency to legacyui-api

**Comment by dkayiwa** [comment]
change from legacyui-omod dependency to legacyui-api

**Comment by mogoodrich** [comment]
Don't know if you are waiting on me on this, but I'm just back from vacation, will review in the next couple days.

File: omod/src/main/java/org/openmrs/module/htmlformentry/web/controller/HtmlFormAjaxValidationController.java [hunk 1]
```diff
@@ -59,11 +59,12 @@ public void duplicateForm(
 		
 		// if the AJAX call has passed a date format, use that, otherwise use the standard date format to parse the date
 		SimpleDateFormat dateFormatter;
-		if (dateFormat != null) {
-			dateFormatter = new SimpleDateFormat(dateFormat);
+		if (dateFormat == null) {
+			dateFormatter = Context.getDateFormat();
+			System.out.println("The default date format is " + dateFormatter.toString());
```
Comment (mogoodrich) [inline omod/src/main/java/org/openmrs/module/htmlformentry/web/controller/HtmlFormAjaxValidationController.java]: Can we remove these system.out.println or change them to log.debug? (There are a few more below as well).

File: omod/src/main/java/org/openmrs/module/htmlformentry/web/controller/HtmlFormAjaxValidationController.java [hunk 2]
```diff
@@ -75,8 +76,10 @@ public void duplicateForm(
 			{
 				Form encForm = enc.getForm();
 				HtmlForm htmlForm = HtmlFormEntryUtil.getService().getHtmlFormByForm(encForm);
+				System.out.println(" Encounter form " + enc.getForm().getFormId() + " encounter date " + enc.getEncounterDatetime() + " passed date " + dateToCheck);
```
Comment (mogoodrich) [inline omod/src/main/java/org/openmrs/module/htmlformentry/web/controller/HtmlFormAjaxValidationController.java]: More System.out.println...

Actually, looking more closely, in the end this file may not need to change at all?

**Comment by ssmusoke** [comment]
@mogoodrich oh my I had missed that, I have cleaned up the commit

**Pull request merged**